llama_hub/sec_filings/base.py

```diff
@@ -0,0 +1,87 @@
+from sec_filings import SECExtractor

```

Comment:
The default way to use llama_hub is now as a package, and so imports should include absolute paths to other files in the same directory: `from llama_hub.sec_filings.sec_filings import SECExtractor`

This will work for now. It won't work with the `download_loader` workflow, but if you want to support that you could do something like 
```python
try:
  from llama_hub.sec_filings.sec_filings import SECExtractor
except ImportError:
  # relative import from file
  from sec_filings import SECExtractor
```

Reply:
Hi @jerryjliu 
Thank you for your feedback
1. The `prepline_sec_filings` is the directory from Unstructured library SEC filings [link](https://github.com/Unstructured-IO/pipeline-sec-filings/tree/main/prepline_sec_filings). This library is useful to get the text from the filings. There are direct pypi package for the repo, hence I decided to just get the folder. I have referenced in the reference section.

Reply:
I incorporated the changes that you asked for in the imports. Also, the `prepline_sec_filings` is essential for my loader as it is useful to get all the texts. As the unstructured library for SEC filings does not have a Pypi package, so importing the folder is useful.

Reply:
i see, thanks